packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -118,7 +118,8 @@ module.exports = {
           // Point ESLint to our predefined config.
           options: {
             configFile: path.join(__dirname, '../.eslintrc'),
-            useEslintrc: false
+            useEslintrc: false,
+            cache: true
```

Comment:
We'll likely want for this to get merged first: https://github.com/MoOx/eslint-loader/pull/151. Or it will be confusing to ejected users who change the rules.